src/Tracer.php

```diff
@@ -73,8 +92,8 @@ private function addUnhandledEvent(Event $event): void
 
     private function addTestPassed(Passed $event): void
     {
-        /** @var TestMethod */
```

Comment:
Please keep this annotation, my editor does not use assertions for type recognition.

Reply:
I believe that [PHP Intelephense](https://marketplace.visualstudio.com/items?itemName=bmewburn.vscode-intelephense-client) use it. I'm using PHPStorm which is using them correctly.